src/plugins/vis_type_timeseries/public/application/components/lib/tick_formatter.js

```diff
@@ -48,7 +49,7 @@ export const createTickFormatter = (format = '0,0.[00]', template, getConfig = n
       }
     }
     try {
-      return render({ value });
+      return value === DEFAULT_VALUE ? value : render({ value });
```

Comment:
let's optimize that code a little and add that check in the beginning of that method. I don't see any reasons to execute `formatter.convert` to return finally the initial `val`.

Reply:
done